### DISCUSSION TITLE
Support for OpenAPI 3.1

### DISCUSSION THREAD

**Discussion by solita-antti-mottonen** [nordic-institute/X-Road#1849] [Ideas]
Any change on supporting OpenAPI 3.1 in Security Server Sidecar when adding a service description? We're trying to add a service, and with our tech stack we can autogenerate OpenAPI 3.1 spec, but not 3.0.

**Comment by raits** [comment]
Hello @solita-antti-mottonen,

Thank you for the question, we will look into adding support for OpenAPI 3.1 with the following backlog item: https://nordic-institute.atlassian.net/browse/XRDDEV-2520

**Comment by raits** [comment]
Hello @solita-antti-mottonen,

We have completed the ticket to support OpenAPI 3.1: https://github.com/nordic-institute/X-Road/pull/1870/files

The support will be included in release 7.4.0, which is scheduled for release later this month.
